apps/prairielearn/src/server.js

```diff
@@ -795,6 +795,17 @@ module.exports.initExpress = function () {
       next();
     },
   );
+  app.use(
+    '/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/access_overrides',
+    [
+      function (req, res, next) {
+        res.locals.navSubPage = 'access_overrides';
+        next();
+      },
+      require('./pages/instructorAssessmentOverrides/instructorAssessmentOverrides'),
```

Comment:
This page should be named `instructorAssessmentAccessOverrides` to match the URL.

Reply:
Do you mean rename the instructorAssesmentOverrides to instructorAssesmentAccessOverrides or change the access_overrides to instructorAssesmentAccessOverrides